Formula/libparc.rb

```diff
@@ -0,0 +1,31 @@
+# Documentation: https://docs.brew.sh/Formula-Cookbook
+#                https://rubydoc.brew.sh/Formula
+# PLEASE REMOVE ALL GENERATED COMMENTS BEFORE SUBMITTING YOUR PULL REQUEST!
```

Comment:
Opening PRs like this one waste CI valuable resources and the time of maintainers who are tasked with reviewing PRs.